classes/blocks/class-sharebutton.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
Not every block needs a PHP file. We only need to add this file once the block needs back end logic (rendering or data fetching). The JS files are sufficient for the block to be usable if it's saved as HTML.